backend/app/routes/broadcasts/addBroadcast.js

```diff
@@ -0,0 +1,24 @@
+const to = require('await-to-js').default;
+const Broadcast = require('../../models/Broadcast');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res) => {
+  const broadcastData = {
```

Comment:
What this is exactly doing? you are copying req.body into broadcastData, how does it help?

Reply:
of course, this doesn't help for now but it may help if we want to pass some data other than req.body to DB  and also makes code more readable.

but yeah I can change it and directly pass it to broadcast.create if you want it that way.

Reply:
I dont think so we will need to add other fields for adding broadcasts

Reply:
ok